tests/python_client/common/common_func.py

```diff
@@ -948,17 +948,18 @@ def gen_normal_expressions():
         "(int64 > 0 && int64 < 400) or (int64 > 500 && int64 < 1000)",
         "int64 not in [1, 2, 3]",
         "int64 in [1, 2, 3] and float != 2",
-        "int64 == 0 || int64 == 1 || int64 == 2",
-        "0 < int64 < 400",
-        "500 <= int64 < 1000",
+        "int64 == 0 || float == 10**2 || (int64 + 1) == 3",
+        "0 <= int64 < 400 and int64 % 100 == 0",
         "200+300 < int64 <= 500+500",
-        "int64 in [300/2, 900%40, -10*30+800, 2048/2%200, (100+200)*2]",
-        "float in [+3**6, 2**10/2]",
-        "(int64 % 100 == 0) && int64 < 500",
-        "float <= 4**5/2 && float > 500-1 && float != 500/2+260",
-        "int64 > 400 && int64 < 200",
```

Comment:
this expression not covered by your modified code, should not be deleted.

Reply:
> this expression not covered by your modified code, should not be deleted.

I have added the line again.